### DISCUSSION TITLE
How to sponsor?

### DISCUSSION THREAD

**Discussion by mastercoms** [piccolo-orm/piccolo#1326] [General]
You are doing a fantastic job, and have gone above and beyond as a maintainer to serve your users. Is there a GitHub Sponsors program available, or any other sort of donation scheme? I see there is a [book sale available here](https://piccolostore.gumroad.com/l/mastering-docs) but I'm having trouble finding a monthly subscription of some sort

**Comment by dantownsend** [comment]
@mastercoms Thanks very much! I need to setup GitHub Sponsors. Thanks for pointing out the book, that does directly help towards the open source projects.
